.travis.yml

```diff
@@ -173,12 +203,19 @@ jobs:
             - libstdc++-5-dev
             - libubsan0
       before_install:
+        - |
+          if [[ "${TRAVIS_EVENT_TYPE}" != "cron" ]]
```

Comment:
We can do better, not starting the job at all, like https://docs.travis-ci.com/user/conditional-builds-stages-jobs/#Conditional-Builds